evm/vm/message.py

```diff
@@ -33,6 +33,7 @@ class Message(object):
     create_address = None
 
     should_transfer_value = None
+    is_static = None
```

Comment:
Why does it default to None? To force all message types to define it?

Reply:
Same convention that I think has raised questions previously.  This is merely a pattern I use to make it easy to know what properties an object has.  

It you have strong feelings about this pattern I'm all ears because it's something that grew organically and I'm not sure it's a good pattern.

Reply:
Not really, I just find it odd for bool attributes to have a default of None, but I think it's more of a gut feeling than anything justifiable